recipes/gocommands/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2010-2023, The Arizona Board of Regents on behalf of The University of Arizona
```

Comment:
When building from source, you don't need to include a copy of the license in the recipe because it can be extracted from the source archive.

Reply:
Thanks for the comment. I'll update the recipe soon!